queue.h

```diff
@@ -29,6 +29,25 @@ typedef struct queue_t {
     pthread_cond_t not_empty;
 } queue_t;
 
+/*
+ * Create a new queue_item_t object.
+ *
+ * This will create and initialize a new queue_item_t and return a handle to
+ * that object.
```

Comment:
Mention in the comment, that this function will return a `NULL` pointer if the `queue_item_t` could not be created.

Reply:
Done.